_pages/fr/index.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: index
+title:  "Activate en Mozilla"
+permalink: /lang/
```

Comment:
since this is the FR version, please already replace "lang" with "fr" (same for the lang: property below)